webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/__snapshots__/controller.test.js.snap

```diff
@@ -19,7 +19,7 @@ exports[`StorageContainer should render controller 1`] = `
     >
       <Select
         allowClear={false}
-        className=""
+        className="controller-type-select"
```

Comment:
if this class is used only for finding the element in the test, I wonder if there is a  better selector, such as by component display name, object property selector or property selector instead of unused css class selectors.
https://airbnb.io/enzyme/docs/api/ReactWrapper/find.html

Reply:
Yeah, it was pretty useless, thanks.